src/components/PaginationRow/PaginationRow.js

```diff
@@ -0,0 +1,223 @@
+              </span>
+              &nbsp;{this.msg('of')}&nbsp;
+              <span className="pagination-pf-items-total">
+                {this.totalCount}
+              </span>
+            </span>
+
+            <ul className="pagination pagination-pf-back">
+              <li className={backButtonsClass}>
+                <a
+                  title={this.msg('firstPage')}
+                  onClick={() => this.setPage(1)}
+                >
+                  <ArrowIcon name="double-left" />
+                </a>
+              </li>
+              <li className={backButtonsClass}>
+                <a
+                  title={this.msg('previousPage')}
+                  onClick={() => this.setPageRelative(-1)}
+                >
+                  <ArrowIcon name="left" />
+                </a>
+              </li>
+            </ul>
+
+            <label className="sr-only">Current Page</label>
+            <input
+              className="pagination-pf-page"
+              value={this.state.pageChangeValue}
+              onChange={e => this.handlePageChange(e)}
+              type="text"
+            />
+            <span>
+              {this.msg('of')}&nbsp;
+              <span className="pagination-pf-pages">{totalPages}</span>
+            </span>
+
+            <ul className="pagination pagination-pf-forward">
+              <li className={nextButtonsClass}>
+                <a
+                  title={this.msg('nextPage')}
+                  onClick={() => this.setPageRelative(1)}
+                >
+                  <ArrowIcon name="right" />
+                </a>
+              </li>
+              <li className={nextButtonsClass}>
+                <a
+                  title={this.msg('lastPage')}
+                  onClick={() => this.setPage(totalPages)}
+                >
+                  <ArrowIcon name="double-right" />
+                </a>
+              </li>
+            </ul>
+          </div>
+        </form>
+      </div>
+    );
+  }
+}
+
+PaginationRow.propTypes = {
+  /** Options for the per page dropdown */
+  perPageOptions: PropTypes.array,
+  /** Current per page setting */
+  perPage: PropTypes.number.isRequired, // eslint-disable-line react/no-unused-prop-types
+  /** Total number of items to paginate */
+  totalCount: PropTypes.number.isRequired,
+  /** Index of page that is currently shown, starting from 1 */
+  currentPage: PropTypes.number.isRequired,
+  /** A callback triggered when the per page dropdown value is selected */
+  onPerPageSet: PropTypes.func,
+  /** A callback triggered when a page is switched */
+  onPageSet: PropTypes.func,
+  /** Strings in the component, see PaginationRow.defaultMessages for details */
+  messages: PropTypes.object,
+  /** Class name for the form element */
+  className: PropTypes.string
+};
+
+PaginationRow.defaultProps = {
+  perPageOptions: [],
+  onPageSet: p => {},
+  onPerPageSet: pp => {},
+  messages: {},
+  className: 'content-view-pf-pagination'
```

Comment:
Passing a `className` will override the `content-view-pf-pagination`, shouldn't it always be there?
Checkout this pr https://github.com/patternfly/patternfly-react/pull/163

You can add another prop called `baseClassName` that can have `content-view-pf-pagination` as default value.
So consumers can add more classes when using `className` and override the base class when passing `baseClassName`.

Reply:
+1